monitoring-alert-rules/README.md

```diff
@@ -4,49 +4,51 @@
 
 This example shows how to configure alert rules in Kyma.
 
-1. Define a new alerting rule in Prometheus.
-2. Configure Alertmanager to handle alerts.
+- Define a new alert rule for AlertManager.
```

Comment:
We cannot have a one-item list. Please convert it into a sentence which suits the previous one.

Reply:
done